app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java

```diff
@@ -80,7 +80,7 @@ private void showChildNotification(Context context, Intent intent, boolean heads
         int priority = headsUp ? NotificationCompat.PRIORITY_HIGH : NotificationCompat.PRIORITY_DEFAULT;
         Intent openIntent = SessionDetailActivity.createIntent(context, sessionId);
         openIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
-        PendingIntent pendingIntent = PendingIntent.getActivity(context, 0, openIntent, 0);
+        PendingIntent pendingIntent = PendingIntent.getActivity(context, 0, openIntent, PendingIntent.FLAG_UPDATE_CURRENT);
```

Comment:
Is it okay to merge this on this PR?
Or will you revert this change and send another PR?